administrator/components/com_associations/controllers/association.php

```diff
@@ -28,10 +28,8 @@ class AssociationsControllerAssociation extends JControllerForm
 	public function cancel($key = null)
 	{
 		JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
-
-		$extension = $this->input->get('extension', '', 'string');
-
-		$key = $extension !== '' ? 'com_categories.category|' . $extension : $this->input->get('acomponent', '') . '.' . $this->input->get('aview', '');
+		
+		$key = $this->input->get('component', '', 'string');
 		$cp  = AssociationsHelper::getComponentProperties($key);
```

Comment:
just
$cp  = AssociationsHelper::getComponentProperties($this->input->get('component', '', 'string'));